awspub/context.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import yaml
 from string import Template
+from typing import Dict
```

Comment:
since we're targeting 3.9+, we can just use `dict[...]` everywhere. no need to import